mne/io/artemis123/utils.py

```diff
@@ -82,3 +84,55 @@ def _compute_mne_loc(coil_loc):
     R = rotation3d_align_z_axis(z_axis)
     loc[3:13] = R.T.reshape(9)
     return loc
+
+
+def _read_pos(fname):
```

Comment:
This looks a lot like ``_read_pos`` from ``mne/io/ctf/eeg.py``. Can you check if you can reuse some part of the code and maybe move the reused part to ``mne/io/utils``?